examples/src/main/java/com/mapbox/navigation/examples/core/MapboxNavigationActivity.kt

```diff
@@ -464,19 +512,23 @@ class MapboxNavigationActivity : AppCompatActivity() {
             Point.fromLngLat(it.longitude, it.latitude)
         } ?: return
 
+        waypoints.add(destination)
```

Comment:
the only waypoint added here is the destination, can we leave it as `listof(origin, destination)` as that is the only thing handled right now?

waypoint logic can get complicated and it doesn't seem like we're adding support for it here